README.adoc

```diff
@@ -79,29 +81,30 @@ Assuming apps have been deployed configuring `spring.cloud.stream.bindings.appli
 
 === Controlling eviction
 
-Internally the collector maintains a cache of the metrics it receives. The default for metric emission is every 5 seconds, but can be tuned on the application by using Spring Boot's metrics exporter scheduling control, please refer to the docs http://docs.spring.io/spring-cloud-stream/docs/Chelsea.SR1/reference/htmlsingle/index.html#_metrics_emitter[here] to configure your applications.
+Internally the collector maintains a cache of the metrics it receives. The default for metric emission is every 60 seconds (for Spring Boot 1.x applications it is every 5 seconds), but can be tuned on the application by using Spring Boot's metrics exporter scheduling control, please refer to the docs https://docs.spring.io/spring-cloud-stream/docs/Elmhurst.RELEASE/reference/htmlsingle/index.html#spring-cloud-stream-overview-metrics-emitter[here] to configure your applications.
 
-The default for the collector is to evict any metric reading that was not updated over the past 30 seconds.
+The default for the collector is to evict any metric reading that was not updated over the past 90 seconds.
```

Comment:
How will this change the UI behavior? We need to update it in the SCDF docs.

Reply:
As explained above if the eviction timeout is lower that the emission interval the UI metrics visualization will became unstable (comes an goes...)